src/index.html

```diff
@@ -2,16 +2,254 @@
+            type="password" 
+            id="password" 
+            name="password"
+            class="input"
+          > 
+        </label>
+      </fieldset>
+      <fieldset class="fieldset third">
+        <legend>An interesting fact about you!</legend>
+        <label 
+          for="cats" 
+        >
+          Do you love cats?
+          <input 
+            type="radio" 
+            id="cats" 
+            name="cats"
+            class="input"
+          >
+          <span>yes</span>
+          <input 
+            type="radio" 
+            id="cat" 
+            name="cat"
+            class="input"
+          >
+          <span>No</span>
+        </label>
+        <label 
+          for="color"
+          class="label"
+        >
+          What is your favorite color?
+          <input 
+            type="color" 
+            id="color" 
+            name="color" class="input"
+          >
+        </label>
+        <label 
+          for="time"
+          class="label"
+        >
+          What time do you go to bed?
+          <input 
+            type="time" 
+            id="time" 
+            name="time" 
+            step="1"
+            class="input"
+          >
+        </label>
+        <label 
+          for="cars"
+          class="label"
+        >
+          What are your favorite brand of cars?
+          <select 
+            name="cars" 
+            id="cars" 
+            multiple
+            class="input"
+          >
+            <option 
+              value="Bmw"
+            >
+              BMW
+            </option>
+            <option 
+              value="Audi"
+            >
+              Audi
+            </option>
+            <option 
+              value="lada"
+            >
+              Lada
+            </option>
+          </select>
+        </label>
+        <label 
+          for="rate"
+          class="label"
+        >
+          How do you rate our work?
+          <input 
+            type="range" 
+            id="rate" 
+            name="rate" 
+            value="0"
+            class="input"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset fourth">
+        <legend>Additional info:</legend>
+        <label 
+          for="comments"
+          class="fourth_label"
+        >
+          comments:
+          <textarea 
+            class="input"
+            id="comments" 
+            name="comments" 
+            minlength="10" 
+            maxlength="150"
+          >
+          </textarea>
+        </label>
+        <label 
+          for="recommend"
+          class="label"
+        >
+          Would you recommend us?
+          <select name="recommend" 
+            id="recommend"
+            class="input"
+          >
+            <option 
+              value="yes"
+            >
+              yes
+            </option>
+            <option 
+              value="no"
+            >
+              no
+            </option>
```

Comment:
yet again, over-stretching code.
```suggestion
            <option value="yes">yes</option>
            <option value="no">no</option>
```